org.eclipse.jface.text/src/org/eclipse/jface/text/reconciler/MonoReconciler.java

```diff
@@ -98,4 +100,20 @@ protected void initialProcess() {
 			extension.initialReconcile();
 		}
 	}
+
+	@Override
+	public void install(ITextViewer textViewer) {
+		super.install(textViewer);
+		if (fStrategy instanceof ITextViewerLifecycle) {
+			((ITextViewerLifecycle) fStrategy).install(textViewer);
+		}
+	}
+
+	@Override
+	public void uninstall() {
+		super.uninstall();
```

Comment:
For the sake of symmetry, you might want to call the strategies uninstall before calling super.uninstall. Same is true for the some of the other impls.

Reply:
fixed